Print_A_Bot/controls/utils/system.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+from logging import getLogger
+from os import path, setpgrp
+
+from django.conf import settings
+
+
+logger = getLogger(__name__)
+
+
+def call_sudo_command(command, new_process=False, **kwargs):
+    """
+    Call django management command with kwargs -- kwargs keys must match command line option names vs var names.
+    """
+    cmd_args = []
+    for k, v in kwargs.iteritems():
+        if type(v) is bool:
+            cmd_args.append('--%s' % k)
+        else:
+            cmd_args.append('--{}={}'.format(k, v))
+    logger.debug('making sudo command call %s %s' % (command, ' '.join(cmd_args)))
+    cmd = ['sudo', 'python', path.join(settings.BASE_DIR, 'manage.py'), command] + cmd_args
+    if new_process:
+        subprocess.Popen(['nohup', ] + cmd,
+                         stdout=open('/dev/null', 'w'),
+                         stderr=open('/var/log/print-a-bot.log', 'a+'),
+                         preexec_fn=setpgrp)
+    else:
+        subprocess.call(cmd)
+
+
+def process_running(name):
+    output = subprocess.check_output(['ps', 'aux', '--cols', '200'])  # prevent ps from trimming columns
```

Comment:
`ps` is a linux command `aux` are values that are passed to it that lists details about all programs/processes that are running